ui/app/templates/vault/cluster/access/control-groups.hbs

```diff
@@ -0,0 +1,32 @@
+<PageHeader as |p|>
+  <p.levelLeft>
+    <h1 class="title is-3">
+      Control Groups
+    </h1>
+  </p.levelLeft>
+  <p.levelRight>
+    {{#link-to 'vault.cluster.settings.control-groups' class="button has-icon-right is-ghost is-compact"}}
+      Configure
+      {{i-con glyph="chevron-right" size=11}}
```

Comment:
`<ICon>` or `{{i-con}}`? Seems like you are using `<ICon>` above

Reply:
Whoops! certainly meant to use angle brackets, it's still new so forgot to do it here 😅. I'll get that fixed up.